cmd/ci-op-configs-mirror/main.go

```diff
@@ -74,6 +74,22 @@ func main() {
 			rbc.CanonicalGoRepository = strP(fmt.Sprintf("github.com/%s/%s", repoInfo.Org, repoInfo.Repo))
 		}
 
+		if rbc.ReleaseTagConfiguration != nil {
+			rbc.ReleaseTagConfiguration.Name = fmt.Sprintf("%s-priv", rbc.ReleaseTagConfiguration.Name)
+			rbc.ReleaseTagConfiguration.Namespace = privatePromotionNamespace
```

Comment:
This will change *all* namespaces to the private namespace, right? We only need to change the `ocp` namespace references, no others.